File: bumperbot_ws/src/bumperbot_localization/nodes/plot_trajectory_node.py
#!/usr/bin/env python3
import rospy
from bumperbot_localization.plot_trajectory import PlotTrajectory


if __name__== '__main__':
    rospy.init_node('plot_trajectories_node')
    plot_ground_truth = PlotTrajectory("base_footprint", "ground_truth_path")
    plot_noisy = PlotTrajectory("base_footprint_noisy", "noisy_path")
    plot_ekf = PlotTrajectory("base_footprint_ekf", "ekf_path")
    
    rospy.spin()

File: bumperbot_ws/src/bumperbot_localization/nodes/plot_trajectory_node.cpp
#include "bumperbot_localization/plot_trajectory.h"


int main(int argc, char **argv)
{
    ros::init(argc, argv, "plot_trajectory_node");
    ros::NodeHandle nh;
    PlotTrajectory plot_ground_truth(nh, "base_footprint", "ground_truth_path");
    PlotTrajectory plot_noisy(nh, "base_footprint_noisy", "noisy_path");
    PlotTrajectory plot_ekf(nh, "base_footprint_ekf", "ekf_path");
    ros::spin();

    return 0;
}